### PR TITLE
Print default when CUDA and CuDNN versions are not specified

### LINKED ISSUE #10456: Default CUDA / cuDNN version not used in template string
When leaving version for CUDA / cuDNN empty, to default to 8.0 / 6, they are not properly used in the template at marks **_blank_**

Please specify the Cuda SDK version you want to use, e.g. 7.0. [Leave empty to default to CUDA 8.0]:
Please specify the location where CUDA **_blank_** toolkit is installed. Refer to README.md for more details. [Default is /usr/local/cuda]:
Please specify the cuDNN version you want to use. [Leave empty to default to cuDNN 6.0]:
Please specify the location where cuDNN **_blank_** library is installed. Refer to README.md for more details. [Default is /usr/local/cuda]:

Instead, there is just a two-space gap.
The script seems to miss a proper fallback for `TF_CUDA_VERSION` / `TF_CUDNN_VERSION` unset.

### PR DESCRIPTION
**Pull request by yifeif**
Fix #10456.

### CODE REVIEW

**Comment by yifeif** [comment]
@tensorflow-jenkins test this please

**Comment by gunan** [comment]
I am OOO, but @caisq should be a better reviewer than me for any bash code.

File: configure [hunk 1]
```diff
@@ -492,6 +492,10 @@ while true; do
   if [ -z "$TF_CUDA_VERSION" ]; then
     read -p "Please specify the CUDA SDK version you want to use, e.g. 7.0. [Leave empty to default to CUDA 8.0]: " TF_CUDA_VERSION
   fi
+  # Set default CUDA version if not set
+  if [ -z "$TF_CUDA_VERSION" ]; then
```
Comment (caisq) [inline configure]: Can be simplified as:
TF_CUDA_VERSION=${TF_CUDA_VERSION:-8.0}

File: configure [hunk 2]
```diff
@@ -587,6 +587,10 @@ while true; do
   if [ -z "$TF_CUDNN_VERSION" ]; then
     read -p "Please specify the cuDNN version you want to use. [Leave empty to default to cuDNN 6.0]: " TF_CUDNN_VERSION
   fi
+  # Set default CUDNN version if not set
+  if [ -z "$TF_CUDNN_VERSION" ]; then
```
Comment (caisq) [inline configure]: As above, can be simplified as:
TF_CUDNN_VERSION=${TF_CUDNN_VERSION:-6}

**Review by caisq — CHANGES_REQUESTED** [review]

**Review by caisq — APPROVED** [review]

**Comment by caisq** [comment]
@tensorflow-jenkins test this please

**Comment by yifeif** [comment]
@tensorflow-jenkins test this please

**Comment by yifeif** [comment]
the xla failure is irrelevant, here is a passing run https://ci.tensorflow.org/job/tensorflow-pull-requests-xla/1743/

**Pull request merged**